modules/ocf_stats/manifests/prometheus.pp

```diff
@@ -0,0 +1,35 @@
+# prometheus daemon config
+class ocf_stats::prometheus {
+  class { '::prometheus':
+    version        => '2.0.0',
+    alerts         => {},
+    scrape_configs => [
+      { 'job_name' => 'prometheus',
+        'scrape_interval' => '10s',
+        'scrape_timeout'  => '10s',
+        'static_configs'  => [
+          { 'targets' => [ 'localhost:9090' ],
+          'labels'  => { 'alias' => 'Prometheus'}
+          }
+        ]
+      },
+      { 'job_name' => 'node',
+        'scrape_interval' => '5s',
+        'scrape_timeout'  => '5s',
+        'static_configs'  => [
+          { 'targets' => [
+            # A list of all hosts to scrape.
+            # We should probably figure out a better way of enumerating all the
+            # hosts.
```

Comment:
would hiera or puppetdb help here? this is a similar problem to the firewall issues of enumerating hosts. Ideally though we wouldn't have to manually specify which hosts to collect stats from.

Reply:
[Querying from PuppetDB](https://puppet.com/docs/puppetdb/5.1/api/query/tutorial.html#querying-with-puppet-code) would probably be the best option.

Reply:
This can be specified in hiera (see the end of [the README](https://forge.puppet.com/puppet/prometheus/readme))-- would that be useful?